odl/solvers/functional/functional.py

```diff
@@ -245,12 +245,12 @@ def bregman(self, point, subgrad=None):
 
         Notes
         -----
-        Given a functional :math:`f`, which has a (sub)gradient
-        :math:`\partial f`, and given a point :math:`y`, the Bregman distance
-        functional :math:`D_f(\cdot, y)` in a point :math:`x` is given by
+        Given a functional :math:`f`, a point :math:`y`, and a (sub)gradient
+        :math:`p \in \partial f(y)`, the Bregman distance functional
+        :math:`D_f^p(\cdot, y)` in a point :math:`x` is given by
 
         .. math::
-            D_f(x, y) = f(x) - f(y) - \\langle \partial f(y), x - y \\rangle.
+            D_f^p(x, y) = f(x) - f(y) - \\langle p, x - y \\rangle.
 
 
         For mathematical details, see `[Bur2016]`_.
```

Comment:
Can you make that an inline link,
```
 `[Bur2016] <https://arxiv.org/abs/1505.05191>`_
```
?

A bit below in `References`, the "Wikipedia article" is just lying around there, I think it would be better to have that in the text above.